### DISCUSSION TITLE
WIN 11 - Audio_sink not taking in new device ID / Name.  Keeps going to default sound device

### DISCUSSION THREAD

**Discussion by Lethoric** [LizardByte/community#542] [Sunshine] [answered]
### Select Topic Area

Question

### Body

Greetings,

I have been stuck for the past hour on this issue, following the guide and online help, but my Sunshine keeps going for the default audio device and not the device I would like it to capture from.

I have tried the following:
audio_sink = SYSTEM (BRIDGE CAST)
audio_sink = {0.0.0.00000000}.{77e0774e-3904-456b-9132-a95890cf2828}
audio_sink = {77e0774e-3904-456b-9132-a95890cf2828}
audio_sink = 77e0774e-3904-456b-9132-a95890cf2828

Sunshine ignores the new values and keeps taking my default speaker.

Any help in this would be appreciated! :)

**Comment by ReenigneArcher** [comment]
Did you save AND restart Sunshine?

**Reply by Lethoric** [reply, marked as answer]
I did, yes.

I was able to find a solution 20 minutes after this post.  I disabled the default audio device, and once I did that, Sunshine was able to pick up and use my System Bridge cast audio device.
